src/main/java/study/carsharing/configuration/SpringConfig.java

```diff
@@ -3,21 +3,31 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import study.carsharing.repository.BoardRepository;
 import study.carsharing.repository.MemberRepository;
+import study.carsharing.service.BoardService;
+import study.carsharing.service.BoardServiceImpl;
 import study.carsharing.service.MemberService;
 import study.carsharing.service.MemberServiceImpl;
 
 @Configuration
 public class SpringConfig {
     private final MemberRepository memberRepository;
+    private final BoardRepository boardRepository;
```

Comment:
You don't need to register bean like this.

If you add annotation like @Service, @Component, @Repository etc, you can use autoWired injection.